config.go

```diff
@@ -47,6 +48,7 @@ func (c *controllerConfiguration) RegisterFlags(flagSet *pflag.FlagSet) {
 	flagSet.DurationVar(&c.ResyncPeriod, "resync-period", 10*time.Hour, "determines the minimum frequency at which watched resources are reconciled")
 
 	flagSet.Var(&hostCredentials{value: &c.HostCredentials}, "host-credentials", "Host and credential pairs in the form hostname=user:password. Use comma separated pairs for multiple hosts")
+	flagSet.StringVar(&c.ManagerRoleName, "manager-role-name", "postgres_role_manager", "Name of the role which will be managing other roles")
```

Comment:
You may want to mark this as required on this level if it is required. So that we don't fail in the middle of `database.go`

flagSet.MarkRequired("manager-role-name")